pkg/plugin/v3/scaffolds/init.go

```diff
@@ -31,15 +31,14 @@ import (
 	"sigs.k8s.io/kubebuilder/v2/pkg/plugin/v3/scaffolds/internal/templates/config/manager"
 	"sigs.k8s.io/kubebuilder/v2/pkg/plugin/v3/scaffolds/internal/templates/config/prometheus"
 	"sigs.k8s.io/kubebuilder/v2/pkg/plugin/v3/scaffolds/internal/templates/config/rbac"
-	"sigs.k8s.io/kubebuilder/v2/pkg/plugin/v3/scaffolds/internal/templates/config/webhook"
 	"sigs.k8s.io/kubebuilder/v2/pkg/plugin/v3/scaffolds/internal/templates/hack"
 )
 
 const (
 	// ControllerRuntimeVersion is the kubernetes-sigs/controller-runtime version to be used in the project
 	ControllerRuntimeVersion = "v0.7.0-alpha.6"
 	// ControllerToolsVersion is the kubernetes-sigs/controller-tools version to be used in the project
-	ControllerToolsVersion = "v0.3.0"
+	ControllerToolsVersion = "2f203e490e8af05e988fdac460151ff3a93f3ee7"
```

Comment:
@estroz an issue is tracked for that: https://github.com/kubernetes-sigs/kubebuilder/issues/1827
And we also have the release of 0.4.1